src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: #{$calendar-width};
+  padding: #{$calendar-padding};
+  gap: 1px;
+  margin: 50px auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.